content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Hardening Guide - Scheduler Configuration"
+description: >
+    Information about how to make the Kubernetes scheduler more secure.
+content_type: concept
+draft: false
+weight: 90
+---
+
+<!-- overview -->
+[kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler) is one of the critical components of the control plane. Generally, it the of the top consideration for the security of the Control Plane. 
+
+In this document we explain the effects of a misconfigured scheduler and how we can make the scheduler more secure.
```

Comment:
(nit)
[Avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).